src/uu/numfmt/src/format.rs

```diff
@@ -145,7 +145,15 @@ fn transform_from(s: &str, opts: &TransformOptions) -> Result<f64> {
     let (i, suffix) = parse_suffix(s)?;
     let i = i * (opts.from_unit as f64);
 
-    remove_suffix(i, suffix, &opts.from).map(|n| if n < 0.0 { -n.abs().ceil() } else { n.ceil() })
+    remove_suffix(i, suffix, &opts.from).map(|n| {
+        if opts.from == Unit::None {
```

Comment:
maybe add a simple comment to explain the why

Reply:
Ok, I added a comment.